sdmetrics/reports/multi_table/diagnostic_report.py

```diff
@@ -0,0 +1,321 @@
+"""Multi table diagnostic report."""
+
+import copy
+import itertools
+import pickle
+import sys
+import warnings
+
+import numpy as np
+import pandas as pd
+import pkg_resources
+import tqdm
+
+from sdmetrics.errors import IncomputableMetricError
+from sdmetrics.multi_table import (
+    BoundaryAdherence, CategoryCoverage, NewRowSynthesis, RangeCoverage)
+from sdmetrics.reports.single_table.plot_utils import (
+    get_column_boundaries_plot, get_column_coverage_plot, get_synthesis_plot)
+from sdmetrics.reports.utils import aggregate_metric_results
+
+RESULT_DETAILS = {
```

Comment:
Is this the same dictionary as in [this file](https://github.com/sdv-dev/SDMetrics/blob/master/sdmetrics/reports/single_table/diagnostic_report.py)? If so can we pull it out